crates/bevy_color/Cargo.toml

```diff
@@ -10,17 +10,19 @@ keywords = ["bevy", "color"]
 rust-version = "1.76.0"
 
 [dependencies]
-bevy_math = { path = "../bevy_math", version = "0.14.0-dev" }
+bevy_math = { path = "../bevy_math", version = "0.14.0-dev", default-features = false }
 bevy_reflect = { path = "../bevy_reflect", version = "0.14.0-dev", features = [
   "bevy",
-] }
+], optional = true }
 bytemuck = { version = "1", features = ["derive"] }
 serde = { version = "1.0", features = ["derive"], optional = true }
 thiserror = "1.0"
 wgpu-types = { version = "0.20", default-features = false, optional = true }
 encase = { version = "0.8", default-features = false }
 
 [features]
+default = []
+bevy = ["dep:bevy_reflect"]
```

Comment:
This feature shouldn't be needed / exist: we can just use `bevy_reflect` directly.

Reply:
Ah thanks, I forget that's a thing. Pushed